wezterm-gui/src/overlay/quickselect.rs

```diff
@@ -258,6 +258,7 @@ impl QuickSelectOverlay {
                 render.dirty_results.add(*pos);
             }
             render.viewport = viewport;
+            render.update_search();
```

Comment:
For the case where no range was set in the args, then I think this will re-search the entire scrollback each time the window is scrolled? That feels undesirable!
Is your goal to update the highlighted text as the scrollback is moved?
If so, I think it might be best to make this conditional on the range arguments and update only if the new viewport is now outside of the last-searched range